python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py

```diff
@@ -182,6 +204,22 @@ def optimize_llm(
         from transformers.models.qwen2.modeling_qwen2 import Qwen2ForCausalLM
         from ipex_llm.transformers.npu_models.qwen2_mp import qwen2_casullm_forward
         convert_forward(model, Qwen2ForCausalLM, qwen2_casullm_forward)
+
+        # for Qwen2-7B-Insturct, divide lm_head into 7 parts
+        if model.config.hidden_size == 3584 and model.config.vocab_size == 152064 and \
+                os.environ.get("IPEX_LLM_CPU_LM_HEAD", "0") == "0":
+            np_dtype = np.uint8 if model.lm_head_0.weight.dtype == torch.uint8 else np.int8
+            split_num = 7
```

Comment:
define a global variable

Reply:
Not sure how to define it as a global variable as different model may have different value ?